docs/recipes/IsolatingSubapps.md

```diff
@@ -0,0 +1,79 @@
+# Isolating Redux Subapps
+
+Consider the case of a "big" app that embeds smaller "subapps":
+
+```js
+import React, { Component } from 'react'
+import Subapp from './subapp/Root'
+
+class BigApp extends Component {
```

Comment:
It’s a bit problematic that throughout the recipe, we use different names: `BigApp`, `Subapp` (different capitalization btw), `App`, `Root`. Can we make them consistent to illustrate the point more vividly?